frontend/common/src/hooks/useCustomEvent.ts

```diff
@@ -0,0 +1,23 @@
+import { ReactPlugin } from "@microsoft/applicationinsights-react-js";
+import { ICustomProperties } from "@microsoft/applicationinsights-web";
+import { useState, useEffect, useRef } from "react";
+
+export default function useCustomEvent(
+  reactPlugin: ReactPlugin,
```

Comment:
Does reactPlugin have to be a parameter?  Could this hook just call `useAppInsightsContext();` itself?

Reply:
it has to call reactPlugin.trackEvent which is not available in that hook. I will give it another look on this

Reply:
this has been resolved now